src/backend/web/routes/post.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { getPost } = require('../../utils/storage');
+
+const post = express.Router();
+
+post.get('/:id', async (req, res) => {
```

Comment:
Are we going to get burned here by URLs that aren't encoded property as the `id`?  A lot of `guid`s for Posts are just URIs or even URLs.  Should we use `/*` and grab everything off the route that way, or are we good if the passed id contains path separators?  I've had issues with this in the past.